addons/mass_mailing_crm/models/crm_lead.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class CrmLead(models.Model):
+    _name = 'crm.lead'
```

Comment:
Do you have to re-specify the name when inheriting?
I guess it doesn't change much, I'm realizing I personally never do it.